README.md

```diff
@@ -1,44 +1,233 @@
-# bridgehead
+# Bridgehead
+
+This repository contains all information and tools to deploy a bridgehead. If you have any questions about deploying a bridgehead, please [contact us](mailto:verbis-support@dkfz-heidelberg.de).
+
+TODO: Insert comprehensive feature list of the bridgehead? Why would anyone install it?
+
+## Requirements
+Before starting the installation process, please ensure that following software is available on your system:
+
+### [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+To check that you have a working git installation, please run
+``` shell
+cd ~/;
+git clone https://github.com/octocat/Hello-World.git;
+cat ~/Hello-World/README;
+rm -rf Hello-World;
+```
+If you see the output "Hello World!" your installation should be working.
+
+### [Docker](https://docs.docker.com/get-docker/)
```

Comment:
ToDo: The Person installing the Software need to be aware that docker 1.20+ is needed